erpnext/selling/doctype/quotation/quotation.py

```diff
@@ -142,6 +142,11 @@ def set_missing_values(source, target):
 		if customer:
 			target.customer = customer.name
 			target.customer_name = customer.customer_name
+		if source.coupon_code:
+			target.coupon_code = source.coupon_code
```

Comment:
coupon_code has no property enabled no_copy, then target can be auto copy from the source. No need to set explicitly.

Reply:
Agreed. Thanks!